Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -177,6 +177,16 @@ open class CarPlayNavigationViewController: UIViewController, BuildingHighlighti
         let template = CPListTemplate(title: alternativesTitle,
                                       sections: variants)
         template.delegate = self
+        
+        if #available(iOS 14.0, *) {
+            let alternativesEmptyVariantsSubtitle = NSLocalizedString("CARPLAY_ALTERNATIVES_EMPTY_SUBTITLE",
```

Comment:
Please run `scripts/extract_localizable.sh` to add the new text to the .strings files

Reply:
ah ha, I ran the script and pushed up the changes. This should be ready for another look!